kolibri/content/api.py

```diff
@@ -25,6 +25,11 @@ class Meta:
         model = models.ContentNode
         fields = ['parent', 'search', 'prerequisite_for', 'has_prerequisite', 'related', 'recommendations_for', 'recommendations']
 
+    def _dm_complex_lookup(self, dm_hashes):
+        if dm_hashes[1]:
+            return Q(stemmed_metaphone__icontains=dm_hashes[0]) | Q(stemmed_metaphone__icontains=dm_hashes[1])
+        return Q(stemmed_metaphone__icontains=dm_hashes[0])
```

Comment:
A comment at the top in here to describe what's happening would be helpful for future readers.